icu4c/source/data/buildtool/comment_stripper.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+
+import io
+
+class CommentStripper(io.BufferedIOBase):
+    def __init__(self, f):
+        self.f = f
+        self.state = 0
+
+    def detach(self):
+        self.f = None
+
+    def read(self, size=-1):
+        bytes = self.f.read(size)
```

Comment:
Is there no higher-level construct that we can build on, like reading lines and/or reading Unicode characters?